crates/csharp/tests/codegen.rs

```diff
@@ -77,6 +85,165 @@ macro_rules! codegen_test {
 }
 test_helpers::codegen_tests!();
 
-fn verify(_dir: &Path, _name: &str) {
-    // TODO?
+fn verify(dir: &Path, name: &str) {
+    #[cfg(all(target_os = "windows", feature = "aot"))]
+    aot_verify(dir, name);
+}
+
+fn aot_verify(dir: &Path, name: &str) {
+    let mut wasm_filename = dir.join(name);
+    wasm_filename.set_extension("wasm");
+
+    fs::write(
+        dir.join("nuget.config"),
+        r#"<?xml version="1.0" encoding="utf-8"?>
+    <configuration>
+        <config>
```

Comment:
I guess this is copied from the runtime version.  I should have removed this really as I don't think there is much benefit and it may improve the CI time without it.

Reply:
Its fine though and we can address them both at the same time.

Reply:
there a bit of duplicate code and this will become more challenging with a different project settings required for mono work.  Let's follow up with a clean up?